content/docs/guides/troubleshooting.md

```diff
@@ -172,3 +172,12 @@ ks version
 ```
 
 If your ksonnet version is lower than what is specified in the [requirements](/docs/guides/requirements), please upgrade it and follow the [guide](/docs/guides/components/ksonnet) to recreate the app.
+
+## Ksonnet on Windows
```

Comment:
Change "Ksonnet" to "ksonnet".
Hint from a tech writer: Even though the word is at the beginning of the sentence, it's still good practice to follow the product's branding choice.